frontend/pages/setting/password.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div>
+    <v-form ref="passwordForm">
+      <UserFormPassword
+        v-model="user.password"
+        noIcon
+        noValidation
+        label="現在のパスワード"
+      />
+
+      <UserFormPassword
+        v-model="user.newpassword"
+        noIcon
+        label="新しいパスワード"
+      />
+
+      <ErrorMessage :message="message" />
+
+      <Btn @click="updatePassword">パスワードを変更する</Btn>
+    </v-form>
+  </div>
+</template>
+
+<script>
+export default {
+  layout: 'setting',
+  data() {
+    return {
+      header: {
+        title: 'パスワードの設定',
+        backUrl: '/setting',
+      },
+      user: {
+        password: '',
+        newpassword: '',
+      },
+      message: null,
+    }
+  },
+  mounted() {
+    this.updateHeader()
+  },
+  methods: {
+    updateHeader() {
+      // タイトルとして使いたい情報を渡す
+      this.$nuxt.$emit('updateHeader', this.header)
+    },
+    // パスワードの更新
+    updatePassword() {
+      this.messages = []
```

Comment:
初期化しているだけであればnullの方がいいと思います。
```suggestion
      this.messages = null
```

Reply:
修正しました。